test/config/database.yml

```diff
@@ -15,7 +15,13 @@ test:
 test_id_server_1:
   <<: *MYSQL
   database: global_uid_test_id_server_1
+  variables:
```

Comment:
Neat. This was added in Rails 4 and I didn’t know 😢 

https://github.com/rails/rails/pull/8346

Reply:
I thought it was super neat too, it makes the implementation way nicer.